toqito/state_props/tests/test_is_abs_ppt.py

```diff
@@ -1,47 +1,52 @@
 """Unit tests for the is_abs_ppt function in toqito.state_props."""
 
+import re
```

Comment:
Is there a reason `re` is imported? I don't see it being used.

Reply:
@vprusso It is being used in a unit test. 

![image](https://github.com/user-attachments/assets/df8aba1e-2d0f-4a78-8da2-d89f924844f9)